FlashDevelop/Dialogs/ShortcutDialog.cs

```diff
@@ -631,31 +747,23 @@ private void DialogClosing(object sender, FormClosingEventArgs e)
         /// <summary>
         /// When the form is closed, applies shortcuts.
         /// </summary>
-        private void DialogClosed(object sender, FormClosedEventArgs e)
+        private void ShortcutDialog_FormClosed(object sender, FormClosedEventArgs e)
         {
-            for (int i = 0; i < this.shortcutListItems.Length; i++) this.shortcutListItems[i].ApplyChanges(); 
+            for (int i = 0; i < this.shortcutListItems.Length; i++)
+            {
+                this.shortcutListItems[i].ApplyChanges();
+            }
             Globals.MainForm.ApplyAllSettings();
```

Comment:
Is it necessary to call `ApplyAllSettings()`? Can it not just call `ApplyAllShortcuts()` in `ShortcutManager`?